src/kernels/execution/cellExecutionMessageHandler.ts

```diff
@@ -398,6 +398,20 @@ export class CellExecutionMessageHandler implements IDisposable {
             this.execution.executionOrder = msg.content.execution_count;
         }
     }
+    private handleCommOpen(msg: KernelMessage.ICommOpenMsg) {
+        this.ownedCommIds.add(msg.content.comm_id);
+        // Check if this is a message for an Output Widget,
+        const state: { _model_module: string } | undefined =
+            // eslint-disable-next-line @typescript-eslint/no-explicit-any
+            typeof msg.content.data === 'object' && msg.content.data ? (msg.content.data['state'] as any) : undefined;
```

Comment:
```suggestion
        // eslint-disable-next-line @typescript-eslint/no-explicit-any
        const state: { _model_module: string } | undefined = msg.content.data?.state || undefined;
```

I’m wondering if this makes sense.

Reply:
Damn it, missed this, will add this in the next PR